base/setup/lib/setuplib.c

```diff
@@ -477,7 +477,7 @@ GetSourcePaths(
 
     /* Check whether the resolved \SystemRoot is a prefix of the image file path */
     // FIXME: commented out to allow installation from USB
-    // if (RtlPrefixUnicodeString(&SystemRootPath, InstallSourcePath, TRUE))
+    /**/if (RtlPrefixUnicodeString(&SystemRootPath, InstallSourcePath, TRUE))/**/
```

Comment:
```suggestion
    if (RtlPrefixUnicodeString(&SystemRootPath, InstallSourcePath, TRUE))
```
By the way... Is the FIXME above addressed already? Does it install from USB now?

Reply:
No, it doesn't, yet (this change is part of a temporary commit that will be removed in the final version of this PR).
The USB install will have to be investigated separately in the future.